colossalai/context/moe_context.py

```diff
@@ -0,0 +1,139 @@
+import torch
+import torch.distributed as dist
+from .parallel_mode import ParallelMode
+
+
+class MoeInfo:
+    """Moe parallelism information, storing parallel sizes and groups.
+    """
+
+    def __init__(self, ep_size: int, dp_size: int):
+        self.ep_size = ep_size
+        self.dp_size = dp_size
+        self.ep_group = None
+        self.dp_group = None
+

```

Comment:
add assert to your consumptions! make sure the program can run in the env out of your expectation.
assert gpc.get_group(ParallelMode.TENSOR) == 1, "You error info"